cdap-ui/README.rst

```diff
@@ -44,40 +41,31 @@ Unix/Linux Flavors
 
 Windows
 -------
+1. Using the command prompt, stop the CDAP SDK::
 
-**Note:** Currently, installation of the UI pack on Windows requires a zip utility - we recommend
-`7zip <http://www.7-zip.org/>`__ or equivalent.
-
-1. Using the command prompt, stop the CDAP SDK
-
-::
-
-  $ cd ``<CDAP_HOME>``
-  $ bin\cdap sdk stop
+  > cd <CDAP_HOME>
+  > bin\cdap sdk stop
 
 2. Open the ``<CDAP_HOME>`` directory in Explorer
-3. Compress the ``ui`` to save a backup, by right clicking on the ``ui`` directory and
+3. Compress the ``ui`` to save a backup, by right-clicking on the ``ui`` directory and
    choosing **Send To** -> **Compressed (zipped) folder**
 4. Delete the ``ui`` directory after the backup is completed
-5. Extract the UI pack (cdap-ui-pack-4.1.0_p1.zip) in the ``<CDAP_HOME>`` directory, by right clicking on the file,
+5. Extract the UI pack (cdap-ui-pack-4.1.0_p1.zip) in the ``<CDAP_HOME>`` directory, by right-clicking on the file,
    choosing **Extract All**, and specifying the path to the ``<CDAP_HOME>`` directory
 6. A new ``ui`` directory should be created
-7. Using the command prompt, start the CDAP SDK
+7. Using the command prompt, start the CDAP SDK::
 
-::
+  > cd <CDAP_HOME>
+  > bin\cdap sdk start
 
-  $ cd ``<CDAP_HOME>``
-  $ bin\cdap sdk start
 
-
-Steps to update Data Preparation capability
+Steps to Update Data Preparation Capability
 ===========================================
-
-1. Upon installing the UI Pack, go to Cask Market
+1. After installing the CDAP UI Pack and restarting CDAP, from within the CDAP UI go to the Cask Market
 2. From the **Solutions** category, follow the steps for the **Data Preparation** solution
-3. Go to Data Preparation by clicking on the CDAP menu and then choosing **Data Preparation**
-4. If newer version of the Data Preparation libraries have been installed, the UI will show an **Update** button.
-5. Click the **Update** button to enable the newer version of Data Preparation.
+3. Go to *Data Preparation* by clicking on the CDAP menu and then choosing *Data Preparation*
+4. If a newer version of the *Data Preparation* libraries has been released, the UI will show an **Update** button
```

Comment:
I'll have to take your word for it, as step 2 could cover that, and I don't have a way to test that...